modules/nw-route-specific-annotations.adoc

```diff
@@ -70,6 +70,34 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 Environment variables cannot be edited.
 ====
 
+.Router timeout variables
+
+`TimeUnits` are represented by a number followed by the unit: `us` *(microseconds), `ms` (milliseconds, default), `s` (seconds), `m` (minutes), `h` *(hours), `d` (days).
+
+The regular expression is: [1-9][0-9]*(`us`\|`ms`\|`s`\|`m`\|`h`\|`d`)
+[cols="3*"]
+|===
```

Comment:
Suggest adding headers for each of these columns.

Reply:
Added headers